common/networking/packets.def

```diff
@@ -488,7 +488,7 @@ PACKET_GAME_INFO = 16; sc, is-info
   UINT16 history_interest_pml;
   /* size limit for cities before they can celebrate */
   UINT8 celebratesize;
-  BOOL changable_tax;
+  BOOL changable_budget;
```

Comment:
@lmoureaux does this break the network protocol?

Reply:
No, names are meaningless for the protocol (only the data layout is meaningful)